packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -160,6 +161,8 @@ export class AtomicCategoryFacet
   @Prop() public injectionDepth = 1000;
   // @Prop() public customSort?: string; TODO: KIT-753 Add customSort option for facet
 
+  @MapProp() public dependsOn: Record<string, string> = {};
```

Comment:
Although logically, changes happen when we change something on the parent facet, not the dependant facet, I still find it more intuitive to define dependencies from the dependant facet (just like JSUI).